src/components/CardItem/CardItem.tsx

```diff
@@ -0,0 +1,47 @@
+import { ROUTES } from '@/constants';
+import { Heading } from '@components/Heading/Heading';
+import { Paragraph } from '@components/Paragraph/Paragraph';
+import { StarRating } from '@components/Rating/Rating';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+interface ICardItemProps {
+  imageLink: string;
+  year: number;
+  id: string;
+  name: string;
+  description: string;
+}
+
+export const CardItem: React.FC<ICardItemProps> = ({
```

Comment:
move this component to components folder to make it global because we should use it in many components